nestkernel/event.h

```diff
@@ -890,9 +913,12 @@ output_stream( T& d, fwit& pos )
   pos += size_uint_t( d );
 }
 
+// The following operator is used to read the information
+// of the GapJEvent from the buffer in Scheduler::deliver_events_
+// The synid can be skipped here as it is stored in a static vector.
```

Comment:
Could you turn this into a Doxygen comment in the place where the operator is declared?